### PR TITLE
[dev-v2.7] Adding code owners file

### PR DESCRIPTION
**Pull request by lucasmlp**
Adding code owners file so that reviewers are automatically added to PRs that are ready to be reviewed.

This is a follow-up of https://github.com/rancher/charts/pull/3397

### CODE REVIEW

**Comment by lucasmlp** [comment]
The CI is failing because [rancher/charts#3559](https://github.com/rancher/charts/pull/3559) hasn't been merged yet.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+CNAME @rancher/mapps
+Makefile @rancher/mapps
+README.md @rancher/mapps
+_config.yml @rancher/mapps
+configuration.yaml @rancher/mapps
+regsync.yaml @rancher/mapps
+index.html @rancher/mapps
+
+# Elemental 
+packages/elemental @rancher/elemental
```
Comment (recena) [inline .github/CODEOWNERS]: @lucasmlp This path does not exist in `dev-v2.7`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+# Rancher Webhook 
+packages/rancher-webhook @rancher/rancher-team-1-neo-dev
+
+# System Upgrade Controller
+packages/system-upgrade-controller @rancher/rancher-team-2-hostbusters-dev
+
+# UI Extension Operator
+packages/ui-plugin-operator @rancher/mapps
+
```
Comment (recena) [inline .github/CODEOWNERS:85]: This path `packages/rancher-windows-gmsa` is missed

File: .github/CODEOWNERS (hunk 1 above)
Comment (lucasmlp) [reply .github/CODEOWNERS]: Fixed

**Review by recena — APPROVED** [review]

**Pull request merged**